src/js/shared/team-form.js

```diff
@@ -34,9 +34,9 @@ exports.createTeamForm = function createTeamForm(defaults = { }) {
     defaults[def] = { value: defaults[def] };
   }
   return createForm({
-    memberB: textField('Member B application ID:', 256, Object.assign({ required: requiredField }, defaults.memberB)),
-    memberC: textField('Member C application ID:', 256, defaults.memberC),
-    memberD: textField('Member D application ID:', 256, defaults.memberD),
+    memberB: textField('B', 256, Object.assign({ required: requiredField, placeholder: 'Enter the Application ID' }, defaults.memberB)),
+    memberC: textField('C', 256, Object.assign({ placeholder: 'Enter the Application ID' }, defaults.memberC)),
```

Comment:
Maybe add a function to make this more readable e.g. `extendDefaults`?

Reply:
Sure—would you be happy to do that?

Reply:
We can make an issue and not have it block.

Reply:
Opened #314